src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/CodeflowChangeAnalyzer.cs

```diff
@@ -90,7 +90,9 @@ public async Task<bool> ForwardFlowHasMeaningfulChangesAsync(string mappingName,
 
         IEnumerable<string> changedFiles = result.GetOutputLines()
             .Where(file => !ignoredFiles.Contains(file))
-            .Select(file => file.Substring(vmrSourcesPath.Length + 1));
+            .Select(file => file.Length < vmrSourcesPath.Length + 1
+                            ? file
+                            : file.Substring(vmrSourcesPath.Length + 1));
```

Comment:
I don't think we can just blind fix it like this.
It needs to reproed to see how it's even possible a file not from under `src/[repo]` has gotten in the diff.
This way you're basically inviting files from non-src/ content (like the `THIRD-PARTY-NOTICES.txt`) to be put with the other repo files.

I think the real fix will be something like:
`.Where(file => !file.StartsWith(vmrSourcesPath))`

before the `Select()`

But first we must understand why is this.